src/main/java/com/gregtechceu/gtceu/client/renderer/machine/FusionReactorRenderer.java

```diff
@@ -77,4 +77,10 @@ public boolean hasTESR(BlockEntity blockEntity) {
     public boolean isGlobalRenderer(BlockEntity blockEntity) {
         return true;
     }
+
+    @Override
+    @OnlyIn(Dist.CLIENT)
+    public int getViewDistance() {
```

Comment:
cant you instead override `getRenderBoundingBox()` similar to how its done in 1.12?

Reply:
![image](https://github.com/GregTechCEu/GregTech-Modern/assets/39562198/7f4353c4-d539-4bbc-b81a-de3e9bfe759e)

Reply:
The ldlib render api does not provide a method :(
For more information, please see the MetaMachineBlockEntity method with the same name